src/Modules/UserProfileModule/components/AccountSetting/AccountSetting.tsx

```diff
@@ -0,0 +1,287 @@
+                        }
+                      }}
+                    />
+                    {image ? (
+                      <div className="d-flex flex-column justify-content-center  rounded-circle  overflow-hidden  align-items-center">
+                        <img src={image} width={120} height={120} alt="" />
+                      </div>
+                    ) : (
+                      <div className="d-flex flex-column justify-content-center align-items-center text-success">
+                        <i className="fa fa-camera text-main fa-2x"></i>
+                      </div>
+                    )}
+                    <span className="text-dark">
+                      {image ? (
+                        <i
+                          onClick={() => {
+                            setImage(null);
+                          }}
+                          className="fa-solid fa-trash-can"
+                        ></i>
+                      ) : (
+                        ""
+                      )}
+                    </span>
+                  </div>
+                </div>
+                {/* col 1 */}
+                <div className="col-md-6 ">
+                  {/* user name input */}
+                  <div className="input-container">
+                    <input
+                      autoComplete="off"
+                      placeholder="Enter your name"
+                      className={`input-field input-theme  ${
+                        errors.userName && "border-danger"
+                      } `}
+                      type="text"
+                      {...register("userName", userNameValidation)}
+                    />
+                    <label
+                      htmlFor="input-field input-theme"
+                      className="input-label"
+                    >
+                      User Name
+                    </label>
+                    <span className="input-highlight"></span>
+                  </div>
+                  <ErrorMessage
+                    errors={errors}
+                    name="userName"
+                    render={({ messages }) => {
+                      return messages
+                        ? Object.entries(messages).map(([type, message]) => (
+                            <p
+                              className="text-start text-danger ps-2"
+                              key={type}
+                            >
+                              {message}
+                            </p>
+                          ))
+                        : null;
+                    }}
+                  />
+
+                  {/* phone input */}
+                  <div className="input-container">
+                    <input
+                      placeholder="Enter your phone"
+                      className={`input-field input-theme ${
+                        errors.phoneNumber && "border-danger"
+                      }`}
+                      type="text"
```

Comment:
This should be of  type tel. This is usefull  when we open the application from  a phone, as it will only show the keypad.